src/server-middleware/api/entities/organisations.js

```diff
@@ -1,8 +1,12 @@
 const getOrganisations = require('../../../cachers/entities/organisations/get');
-const { errorHandler } = import('../');
+import { errorHandler } from '../';
 
 export default (options = {}) => (req, res) => {
-  getOrganisations({
+  if (!options.redis.url) {
```

Comment:
this breaks the unit test. Just needs to be stubbed there.

Reply:
Actually, it was the removal of prettification of the JSON that broke it. Fixed, and apologies for not testing first.